### DISCUSSION TITLE
Legality

### DISCUSSION THREAD

**Discussion by SSouper** [ObscenityIB/creeper#14] [Q&A]
Has anyone gotten some "interesting" emails from ISPs asking why tf you're scanning their network?
I'm scanning the entire ipv4 address space(with exclude.conf of course)
What are some good VPS hosts for this? I'm currently on scaleway i have no idea if they mind me scanning IPs using their services or not.

I'm trying to build a "player tracker" website where you can search for uuids/usernames and find unlisted servers.

**Comment by ObscenityIB** [comment]
A lot of them dont like "port scanning", but you could argue that since this program only connects on one single port, it is not port scanning.

If you need a VPS that wont punish you, you can google "bulletproof vps" and look for one that allows scanning.
Most that show up under "bulletproof" are for torrenting and hosting copyright information, since they ignore DMCA takedowns.
